File: src/main/java/com/Clair/Hello.java
package com.Clair;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello");
        Person p = new Person();
        p.hello();
        int age = 19; //基本資料型態
        Integer age2 = 19; // age2變物件 (參照資料型態)
        // age2.byteValue()
        char c = 'A';
        byte b = 120;
        float weight = 66.5f; // 預設為double,使用float後須加f
        double height = 1.7;
        boolean adult = true;
        boolean enroll = false;
        String name = "Claire"; //字串字首需大寫
        //為基本資料設計的類別稱包裝類別 Wrapper class
    }
}

File: src/main/kotlin/com/Clair/Hello.kt
package com.Clair

fun main() {
//println("Hello")

// Human().hello()

    val h = Human()
    h.hello()
    //kotlin 沒有基本資料型態, 只有物件所表示的資料型態
    //var 變數 //val 定義後不能改變的值
    val age : Int = 19
    //val age = 19 // kotlin可變數型態自動推斷  ((python..
    // age = 19  // 以val宣告 不可變動值
    var weight = 66.5
    weight = 55.0 //以var宣告可更動值
    var name : String
    name = "Clair"
}

//class函數不能跟Java同名
class Human {
    fun hello(){
        println("Hello Kotlin")

    }
}